lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -132,9 +132,9 @@ pyasn1==0.5.1 ; python_version >= "3.8" and python_version < "3.12"
 pycparser==2.21 ; python_version >= "3.8" and python_version < "3.12"
 pycryptodome==3.19.1 ; python_version >= "3.8" and python_version < "3.12"
 pydantic-tes==0.1.5 ; python_version >= "3.8" and python_version < "3.12"
-pydantic==1.10.13 ; python_version >= "3.8" and python_version < "3.12"
-pydantic[email]==1.10.13 ; python_version >= "3.8" and python_version < "3.12"
-pydot==2.0.0 ; python_version >= "3.8" and python_version < "3.12"
+pydantic==2.5.3 ; python_version >= "3.8" and python_version < "3.12"
+pydantic[email]==2.5.3 ; python_version >= "3.8" and python_version < "3.12"
+pydot==1.4.2 ; python_version >= "3.8" and python_version < "3.12"
```

Comment:
Why reverted back to 1.4.2, merge issue?

Reply:
yep, thanks!